src/base.jl

```diff
@@ -184,7 +184,7 @@ PowerSystem(; buses = [Bus()],
 function PowerSystem(ps_dict::Dict{String,Any}; kwargs...)
         Buses, Generators, Storage, Branches, Loads, LoadZones, Shunts, Services = ps_dict2ps_struct(ps_dict)
         sys = PowerSystem(Buses, Generators,Loads,Branches,Storage,ps_dict["baseMVA"]; kwargs...);
-        return sys
+        return sys , Services
```

Comment:
Are we sure we want to return `sys` and `Services` here? This is going to make the return type across the various constructors type unstable. If you absolutely want to do this, you can create a new function `function get_powersystem_and_services(args...) end` and `return sys, Services`.